geoportailv3/static/js/mymaps/mymaps.html

```diff
@@ -50,43 +50,82 @@
 </div>
 
 <!-- Modification Modal -->
-<ngeo-modal ngeo-modal-shown="ctrl.modifying">
+<ngeo-modal ngeo-modal-shown="ctrl.modifying || ctrl.creating">
```

Comment:
So as I understand, I should create a enum: 
````
/**
* @enum {number}
*/
app.modalType = {
  HIDDEN: 0,
  CREATE: 1,
  MODIFY: 
};
...
/**
* @type {app.modalType}
* @export
**/
this.modal = app.modalType.HIDDEN;
````
Which I can then set `this.modal = app.modalType.CREATE`